editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -117,8 +151,24 @@ export const MembersTable = ({
           </TableBody>
         </Table>
       </TableContainer>
-      {showModal && (
-        <AddNewEditorModal showModal={showModal} setShowModal={setShowModal} />
+      {showAddModal && (
+        <AddNewEditorModal
+          showModal={showAddModal}
+          setShowModal={setShowAddModal}
+          initialValues={initialValues}
+          actionType={"add"}
+        />
+      )}
+      {showUpdateModal && (
+        <AddNewEditorModal
+          showModal={showUpdateModal}
+          setShowModal={setShowUpdateModal}
```

Comment:
Example of a specific `setShowModal` being passed down depending on if it's an "edit" modal